csharp/LogicalColumnStream.cs

```diff
@@ -59,6 +56,6 @@ private static List<Schema.Node> GetSchemaNode(Schema.Node node)
         protected readonly short[]? DefLevels;
         protected readonly short[]? RepLevels;
 
-        protected readonly Schema.Node[]? ArraySchemaNodes;
+        protected readonly Schema.Node[]? SchemaNodesPath;
```

Comment:
Hmm yes the old name isn't really correct any more so it would be nice to keep the new name. Given it's readonly we could possibly keep the old member around but document it as deprecated if we were concerned about backwards compatibility, but as we're bumping the major version I think this is probably fine to change as long as we document the change in the release notes. I'd also be surprised if anybody was inheriting from LogicalColumnStream and was affected by this.